ssl/ssl_lib.c

```diff
@@ -7533,18 +7533,6 @@ void ssl_evp_cipher_free(const EVP_CIPHER *cipher)
     }
 }
 
-const EVP_MD *ssl_evp_md_fetch(OSSL_LIB_CTX *libctx,
-                               int nid,
-                               const char *properties)
-{
-    const EVP_MD *md;
-
-    ERR_set_mark();
-    md = EVP_MD_fetch(libctx, OBJ_nid2sn(nid), properties);
-    ERR_pop_to_mark();
-    return md;
-}
-
 int ssl_evp_md_up_ref(const EVP_MD *md)
```

Comment:
Please also remove the ssl_evp_md_up_ref() and ssl_evp_md_free() calls. Also the equivalent cipher up_ref and free calls can be removed.